rx-java/rxgrpc-stub/pom.xml

```diff
@@ -28,6 +28,7 @@
         <dependency>
             <groupId>io.grpc</groupId>
             <artifactId>grpc-stub</artifactId>
+            <version>${grpc.version}</version>
```

Comment:
This gets picked up from the parent pom's dependency management section.

Reply:
Added version line.